backend/parser.py

```diff
@@ -49,7 +49,7 @@ def add_point(self, timestamp, data: list, latency=None):
         else:
             self.timestamp_diffs.append(0)
 
-        if latency:
+        if latency != None:
```

Comment:
The "!= None" is redundant and should be removed.

Reply:
I found an edge case where adding a point with latency == 0 would cause `if latency:` to evaluate to False and so 0 latency points couldn't be added. Is that a valid edge case or is 0 latency physically impossible?

Reply:
Latency of 0 should be impossible. What you could do instead is initially set latency to -1, and have the condition be "if latency >= 0"

Reply:
I thought about this a bit more. While negative latency *should* be impossible, there is a chance that there could be a bug with timestamping that would give the appearance of negative latency. I think your approach makes sense, though just add a comment stating why the "!= None" is needed.